packages/salesforcedx-vscode-apex/src/views/testRunner.ts

```diff
@@ -77,7 +79,20 @@ export class ApexTestRunner {
   }
 
   public getTempFolder(): string {
-    return os.tmpdir();
+    if (vscode.workspace && vscode.workspace.workspaceFolders) {
+      const workspaceRootPath = path.join(
```

Comment:
Could you rename this variable (maybe to something like 'sfdxDir')?

Reply:
Definitely!